accern_xyme/accern_xyme.py

```diff
@@ -741,6 +741,78 @@ def get_settings(self) -> Dict[str, Any]:
         assert self._settings is not None
         return self._settings
 
+    def get_timing(self, waiting: str) -> Dict[str, Any]:
```

Comment:
`blacklist: List[str]` instead of `waiting: str`

Reply:
also, use the return types I gave you:
```
FnTiming = TypedDict('FnTiming', {
   "name": str,
   "quantity": int,
   "total": float,
   "avg": float,
})
NodeTiming = TypedDict('NodeTiming', {
   "node_avg": float,
   "node_total": float,
   "fns": List[FnTiming],
})
TimingResult = TypedDict('TimingResult', {
    "total": float,
    "nodes": Dict[str, NodeTiming],
})
```
there is no need for `Any` to show up in this code